spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

```diff
@@ -794,6 +800,36 @@ List<Trade> fetchByActionNamedQuery(String action);
 
 This allows table names evaluated with SpEL to be used in custom queries.
 
+==== Projections
+
+Spring Data Spanner supports projections. You can define projection interfaces based
+on domain types and add query methods that return them in your repository:
+
+[source, java]
+----
+public interface TradeProjection {
+
+	String getAction();
+
+	@Value("#{target.symbol + ' ' + target.action}")
+	String getSymbolAndAction();
+}
+
+public interface TradeRepository extends SpannerRepository<Trade, Key> {
```

Comment:
This shouldn't be a `SpannerRepository`, but a `CrudRepository` unless I'm missing something.

Reply:
Both work, but `SpannerRepository` would give them some Spanner-specific methods.